@navikt/core/react/src/form/combobox/FilteredOptions/filtered-options-util.test.ts

```diff
@@ -0,0 +1,18 @@
+import { describe } from "vitest";
+import filteredOptionsUtil from "./filtered-options-util";
+
+describe("filtered options util", () => {
```

Comment:
Only added a test for the code I changed. Could add tests for the rest of the file as well?

Reply:
I don't think we need tests for such simple functions.